cc3d/CompuCellSetup/persistent_globals.py

```diff
@@ -198,10 +199,19 @@ def timestamp_string(self) -> str:
 
         current_time = time.localtime()
         str_f_time = time.strftime
-        timestamp_str = "_" + str_f_time("%m", current_time) + "_" + str_f_time("%d", current_time) + "_" + str_f_time(
-            "%Y", current_time) + "_" + str_f_time("%H", current_time) + "_" + str_f_time("%M",
-                                                                                          current_time) + "_" + str_f_time(
-            "%S", current_time)
+        # timestamp_str = "_" + str_f_time("%m", current_time) + "_" + str_f_time("%d", current_time) + "_" + str_f_time(
+        #     "%Y", current_time) + "_" + str_f_time("%H", current_time) + "_" + str_f_time("%M",
+        #                                                                                   current_time) + "_" + str_f_time(
+        #     "%S", current_time)
+        timestamp_str = "_{}_{}_{}_{}_{}_{}_#{}".format(
```

Comment:
Looks good. I only suggest using f-strings (for minorly improved clarity) and dropping the "#".